spec/directions/bake_numbered_exercise/main_spec.rb

```diff
@@ -5,19 +5,13 @@
 RSpec.describe Kitchen::Directions::BakeNumberedExercise do
   it 'calls v1' do
     expect_any_instance_of(Kitchen::Directions::BakeNumberedExercise::V1).to receive(:bake)
-      .with(exercise: 'exercise1', number: '1.1', suppress_solution: false, note_suppressed_solutions: false)
-    described_class.v1(exercise: 'exercise1', number: '1.1', suppress_solution: false, note_suppressed_solutions: false)
+      .with(exercise: 'exercise1', number: '1.1', suppress_solution_if: false, note_suppressed_solutions: false)
```

Comment:
Also made the same change as I was written in the above comment in the spec file.